lib/custom_lib/Kconfig

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2021, Legrand North America, LLC.
+# SPDX-License-Identifier: Apache-2.0
+
+config CUSTOM_LIB
+	bool "custom_lib Support"
+	help 
+	  This option enables the custom_lib library
+
+if CUSTOM_LIB
+
+config CUSTOM_LIB_GET_VALUE_DEFAULT
+	int "custom_lib_get_value() default return value"
```

Comment:
Since there is a single option, the convention is usually to use `depends` and not `if/else`

Reply:
Accepted, and manually rolled in.